scripts/deploy.sh

```diff
@@ -0,0 +1,145 @@
+    fi
+done
+
+echo "* Testing the connection"
+oc -n openshift-console get routes console -o jsonpath='{.status.ingress[0].routerCanonicalHostname}' >/dev/null
+if [ $? -ne 0 ]; then
+    echo "**ERROR**: Make sure you are logged into an OpenShift Container Platform before running this script"
+    exit 2
+fi
+
+echo "* Checking cluster setup"
+mch_name="multiclusterhub" # TODO: search for name
+oc get mch -n "$INSTALL_NAMESPACE" $mch_name
+if [ $? -ne 0 ]; then
+  echo "**ERROR**: multiclusterhub not installed, please install it before!"
+  exit 3
+fi
+
+# check RHACM version
+major_version=$(oc get MulticlusterHub $mch_name -n "$INSTALL_NAMESPACE" -o jsonpath="{.status.currentVersion}" | cut -d'.' -f 1-2)
+if [ "$major_version" != "2.6" ]; then
+    echo "**ERROR**: This script applies currently only to ACM 2.6!"
+    exit 3
+fi
+
+# launch uninstall if requested
+if [ $uninstall -eq 1 ]; then
+    uninstaller
+    exit 0
+fi
+
+# check that secret file exists
+if [ ! -f $EXPECTED_SECRET_FILE_NAME ]; then
+  echo "**ERROR**: Secret file $EXPECTED_SECRET_FILE_NAME not found in the current directory!"
+  exit 4
+fi
+# and that it is named correctly
+secret_name=$(yq eval '.metadata.name' $EXPECTED_SECRET_FILE_NAME)
+if [ "$secret_name" != $EXPECTED_SECRET_NAME ]; then
+  echo "**ERROR**: Secret in $EXPECTED_SECRET_FILE_NAME must be named $EXPECTED_SECRET_NAME!"
+  exit 5
+fi
+
+# check that CR is there
+if [ ! -f "$CR_PATH" ]; then
+    echo "**ERROR**: Custom Resource $CR_PATH not found!"
+    exit 6
+fi
+# and that it is named correctly
+cr_name=$(yq eval '.metadata.name' "$CR_PATH")
+if [ "$cr_name" != $EXPECTED_CR_NAME ]; then
+  echo "**ERROR**: Custom Resource in $CR_PATH must be named $EXPECTED_CR_NAME!"
+  exit 7
+fi
+
+# check the operator is not already installed
+EXIT_CODE=0
+oc get catalogsources.operators.coreos.com -n "$INSTALL_NAMESPACE" search-v2-operator-catalog > /dev/null 2>&1  || EXIT_CODE=$?
+if [ $EXIT_CODE -eq 0 ]; then
+    # we could try to upgrade if we have a different version, keep it simple for now
+    echo "**ERROR**: operator already installed, please uninstall it before with \"$0 -u\"!"
+    exit 8
+fi
+echo "All good, proceeding with install"
+
+# disable search v1
+echo "* Disable search v1"
+oc patch mch ${mch_name} -n "$INSTALL_NAMESPACE" --type=merge -p '{"spec":{"overrides":{"components":[{"name":"search","enabled": false}]}}}'
+
+echo "* Apply quay.io secret"
+oc apply -f quay_secret.yaml -n "$INSTALL_NAMESPACE"
+
+# deploy operator
+echo "* Deploy search v2"
+# compute version
+printf "Find snapshot tags @ https://quay.io/repository/stolostron/search-operator-bundle?tab=tags\nEnter SNAPSHOT TAG: (Press ENTER for default: ${DEFAULT_SNAPSHOT})\n"
+read -r SNAPSHOT_CHOICE
+if [ "${SNAPSHOT_CHOICE}" == "" ]; then
+    SNAPSHOT_CHOICE=${DEFAULT_SNAPSHOT}
+fi
+operator-sdk run bundle quay.io/stolostron/search-operator-bundle:${DEFAULT_SNAPSHOT} --pull-secret-name $EXPECTED_SECRET_NAME -n "$INSTALL_NAMESPACE"
```

Comment:
Is this correct `operator-sdk run bundle quay.io/stolostron/search-operator-bundle:${DEFAULT_SNAPSHOT}` , or should this be `operator-sdk run bundle quay.io/stolostron/search-operator-bundle:${SNAPSHOT_CHOICE}`

Reply:
yep, switched to use ${SNAPSHOT_CHOICE}, thanks!